agents/a2a_mcp_integration.py

```
@@ -16,246 +16,272 @@
+                "priority": self.priority.value,
+                "strategy": self.transport_strategy.value,
+                "deadline_ms": self.deadline_ms,
+                "requirements": self.performance_requirements,
+            },
         }
-    
+
     @classmethod
-    def from_dict(cls, data: Dict[str, Any]) -> 'A2AMCPMessage':
-        a2a_msg = A2AMessage.from_dict(data['a2a'])
+    def from_dict(cls, data: Dict[str, Any]) -> "A2AMCPMessage":
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)